c2corg_ui/static/partials/cards/outings.html

```diff
@@ -10,10 +10,13 @@
     <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{::locale.lang}})</span>
     <br>
     <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left"></div>
+         uib-tooltip="{{'quality'| translate}} : {{::doc['quality'] | translate}}" tooltip-placement="left">
+    </div>
 
-    <span class="elevation-max" ng-if="::doc['elevation_max']" uib-tooltip="{{'elevation_max' | translate}}">
-      <span translate class="value-title glyphicon glyphicon-sort-by-attributes"></span>
+    <!--elevation-max is shown only in the home feed-->
+    <span class="elevation-max" ng-if="::doc['elevation_max'] && cardCtrl.currentLocation == '/'"
+          uib-tooltip="{{'elevation_max' | translate}}">
+      <span translate class="value-title glyphicon glyphicon-sort-by-attributes rotate-180"></span>
```

Comment:
rotation rotates the glyphicon icon, because it's upside down originally :P

oops, forgot to remove the translate directive!